include/rocksdb/options.h

```diff
@@ -767,6 +767,14 @@ struct DBOptions {
 
   // Allow the OS to mmap file for reading sst tables.
   // Not recommended for 32-bit OS.
+  // When the option is set to true and compression is disabled, the blocks
+  // will not be copied and will be read directly from the mmaped memory
```

Comment:
Not sure this will be in a dictionary but in place of "mmaped" I prefer one of:
* mmapped (using the pattern "map" -> "mapped")
* mmap-ed
* mmap'd